docs/source/reference/routines.rst

```diff
@@ -26,3 +26,16 @@ These functions cover a subset of
    sorting
    statistics
    ext
+
+
+CUB backend for reduction routines
+----------------------------------
+Some CuPy reduction routines, including :func:`~cupy.sum`, :func:`~cupy.min`, :func:`~cupy.max`,
+:func:`~cupy.argmin`, :func:`~cupy.argmax`, and other functions built on top of them, can be
+accelerated by switching to the `CUB backend`_. The switch can be toggled on or off at runtime
+by setting the bool :data:`cupy.cuda.cub_enabled`, which is set to ``False`` by default. Note
```

Comment:
Could you mention that the result may become non-deterministic? That's the primary reason why we have this option.

Reply:
> the result may become non-deterministic

Really? I thought we have all the tests passed (please ignore my noise in the other PR earlier today 😞)?